src/Http/RequestTransformer.php

```diff
@@ -48,6 +48,12 @@ public static function toCake(PsrRequest $request)
             $post = Hash::merge($post, $files);
         }
 
+        $input = null;
+        $stream = $request->getBody();
+        if ($stream->getSize()) {
```

Comment:
I don't completely understand. Cake\Network\Request expects a string for the 'input' if its being injected, which is what this is trying to do.